### DISCUSSION TITLE
Can't Download the Launcher

### DISCUSSION THREAD

**Discussion by DotaChan** [ppy/osu#28283] [Q&A] [answered]
### Type

Other

### Bug description

Every time I try to download the installer it get stuck at at 95% and it stops downloading
And when I try to resume it asks me for a password of something
https://imgur.com/a/JAZ21yG

### Screenshots or videos

non

### Version

currnet

### Logs

non

**Comment by peppy** [comment, marked as answer]
Something is very wrong with your connection, out of our control.
